scarlet2/interpolation.py

```diff
@@ -266,18 +266,38 @@ def body_fun_y(i, args):
 
     return res.reshape(warp[..., 0].shape)
 
-def resample_ops(kimage, shape_in, shape_out, res_in, res_out, interpolant=Quintic()):
+def resample_ops(kimage, shape_in, shape_out, res_in, res_out, phi=None, flip_sign=None, interpolant=Quintic()):
     """
     Resampling operation used in the Multiresolution Renderers
     This is assuming that the signal is Hermitian and starting at 0 on axis=2,
     i.e. f(-x, -y) = conjugate(f(x, y))
     """
+    
+    # Apply rescaling to the frequencies
+    # [0, Fe/2+1]
+    # [-Fe/2+1, Fe/2]
     kcoords_out = jnp.stack(jnp.meshgrid(
-        jnp.linspace(0, shape_in/2 / res_out*res_in, shape_out//2+1),
-        jnp.linspace(-shape_in/2/res_out*res_in, shape_in/2/res_out*res_in, shape_out)
-        ),
-        -1)
+        jnp.linspace(0, 
+                     shape_in / 2 / res_out * res_in, 
+                     shape_out // 2 + 1),
+        jnp.linspace(-shape_in / 2 / res_out * res_in, 
+                     shape_in / 2 / res_out * res_in, 
+                     shape_out)
+        ), -1)
+
     
+    # Apply rotation to the frequencies
+    if phi is not None:
+        R = jnp.array([[jnp.cos(phi), jnp.sin(phi)],
+                       [-jnp.sin(phi), jnp.cos(phi)]])
+        
+        b_shape = kcoords_out.shape
+        kcoords_out = (R @ kcoords_out.reshape((-1, 2)).T).T.reshape((b_shape))
+
+    # TODO: Apply flip on rotation if any
+    # if flip_sign is not None:
+    #     kcoords_out = kcoords_out*flip_sign.reshape((2,1,1))
```

Comment:
Do we still need this sign flip piece. If not, please remove. But also see the next comment